python/setup_project/create_storage_dialog.py

```diff
@@ -33,7 +32,6 @@ def __init__(self, existing_storage_names, parent=None):
         """
 
         super(CreateStorageDialog, self).__init__(parent)
-        self.setWindowFlags(QtCore.Qt.Popup)
```

Comment:
Why was this line present at the first place? Did you dig up a bit the history?

Reply:
Found a better way to handle this, removing `setWindowFlags` will show the storage widget as a Dialog rather than an actual Popup.

My latest change includes a call to `.show()` after editing widget's flags, therefore the widget remains as a popup, and is tied as a children of the admin ui window.